src/popup.ts

```diff
@@ -2,31 +2,38 @@ import './popup.scss';
 
 import mdcAutoInit from '@material/auto-init';
 import {MDCCheckbox} from '@material/checkbox';
-import {MDCChipSet} from '@material/chips/deprecated';
 import {MDCFormField} from '@material/form-field';
 import {MDCMenu} from '@material/menu';
 import {MDCRipple} from '@material/ripple';
-import {MDCSelect} from '@material/select';
 import {MDCSnackbar} from '@material/snackbar';
-import {MDCTextField} from '@material/textfield';
 import {MDCTopAppBar} from '@material/top-app-bar';
 import {ANALYTICS} from './google-analytics';
 import {
-  PopupFormData,
+  type PopupFormData,
   setCheckboxState,
   setInputValue,
   setupGlobalEventLogging,
   setupStorage,
 } from './utils';
+import '@material/web/icon/icon.js';
+import '@material/web/button/filled-button.js';
+import '@material/web/iconbutton/icon-button.js';
+import '@material/web/textfield/filled-text-field.js';
+import '@material/web/checkbox/checkbox.js';
+import '@material/web/select/filled-select.js';
+import '@material/web/select/select-option.js';
+import '@material/web/chips/chip-set';
+import '@material/web/chips/filter-chip';
+
+import {type MdFilledSelect} from '@material/web/select/filled-select.js';
+import {type MdFilledTextField} from '@material/web/textfield/filled-text-field';
+import {type MdCheckbox} from '@material/web/checkbox/checkbox.js';
+import {type MdChipSet} from '@material/web/chips/chip-set';
+import type {MdFilterChip} from '@material/web/chips/filter-chip';
 
```

Comment:
why is type inside the braces on one line but outside the next? are some of these imports (e.g. from "[...]chip-set") duplicates?

Reply:
My bad should have kept that consistent (why does the formatter not do this?
There are no duplicates